platform/darwin/src/MGLStyle.h

```diff
@@ -189,7 +189,7 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 /**
  A set containing the style’s sources.
  */
-@property (nonatomic, strong) NS_MUTABLE_SET_OF(MGLSource *) *sources;
+@property (nonatomic, strong) NS_SET_OF(__kindof MGLSource *) *sources;
```

Comment:
Should we mention that it is possible to get a mutable copy as noted in https://github.com/mapbox/mapbox-gl-native/issues/7459#issuecomment-267810124?